fvm/evm/handler/addressAllocator.go

```diff
@@ -11,6 +11,14 @@ import (
 
 const ledgerAddressAllocatorKey = "AddressAllocator"
 
+var (
+	// prefixes:
+	// the first 12 bytes of addresses allocation
+	// leading zeros helps with storage and all zero is reserved for the EVM precompiles
+	FlowEVMPrecompileAddressPrefix = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}
+	FlowEVMCOAAddressPrefix        = []byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2}
```

Comment:
Maybe make these arrays
```suggestion
	FlowEVMPrecompileAddressPrefix = [...]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1}
	FlowEVMCOAAddressPrefix        = [...]byte{0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 2}
```